org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoWindow.java

```diff
@@ -1066,7 +1067,11 @@ protected String getSQLWhere() {
 		
 		StringBuilder builder = new StringBuilder();
 		MTable table = MTable.get(Env.getCtx(), infoWindow.getAD_Table_ID());
-		if (!hasIsActiveEditor() && table.get_ColumnIndex("IsActive") >=0 ) {
+		MReference ref = m_gridfield != null ? MReference.get(Env.getCtx(), m_gridfield.getAD_Reference_Value_ID()) : null;
+		boolean onlyActive = (ref == null 
+				|| Util.isEmpty(ref.getShowInactive()) 
+				|| MReference.SHOWINACTIVE_No.equalsIgnoreCase(ref.getShowInactive()));
```

Comment:
it is probably more readable and less error prone to encapsulate this as a new method in MReference, i.e `boolean onlyActive = ref == null || !ref.isShowInactiveRecords();`